src/shared/constants.ts

```diff
@@ -77,6 +77,13 @@ export const ecsExecToolkitGuideUrl: string =
 export const ecsRequiredPermissionsUrl: string =
     'https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-exec.html#ecs-exec-enabling-and-using'
 
+// URL for stuck CFN deployment help
+export const stuckCfnDeploymentUrl =
+    'https://aws.amazon.com/premiumsupport/knowledge-center/cloudformation-stack-stuck-progress/'
+
+// URL for AWS service health status
+export const awsHealthStatusUrl = 'https://status.aws.amazon.com/'
```

Comment:
AWS has a personalized health status  https://phd.aws.amazon.com/phd/home#/account/dashboard/open-issues . But I looks like https://status.aws.amazon.com/ redirects to that if your browser is logged-in. 

So https://status.aws.amazon.com/  is probably fine, although it won't prompt to log in.